core/domain/suggestion_services.py

```diff
@@ -634,39 +688,49 @@ def get_all_user_ids_who_are_allowed_to_review(score_category):
         list(str). All user_ids of users who are allowed to review in the given
         category.
     """
-    return [model.user_id for model in
-            user_models.UserContributionScoringModel
-            .get_all_users_with_score_above_minimum_for_category(
-                score_category)]
-
+    return [
+        model.user_id for model in user_models.UserContributionScoringModel
+        .get_all_users_with_score_above_minimum_for_category(score_category)
+    ]
 
-def increment_score_for_user(user_id, score_category, increment_by):
-    """Increment the score of the user in the category by the given amount.
 
-    In the first version of the scoring system, the increment_by quantity will
-    be +1, i.e, each user gains a point for a successful contribution and
-    doesn't lose score in any way.
+def create_new_user_scoring(user_id, score_category, score):
```

Comment:
Shouldn't this be encapsulated within some get_user_scoring() function, rather than called directly? That function should either return the contents of the storage model, or an empty object if the storage model does not exist.